FrontEnd/src/components/SignUp/components/signup/signup-form/SignUpFormContent.js

```diff
@@ -1,24 +1,17 @@
-import { useEffect, useState } from 'react';
+import React, { useEffect, useState ,  Suspense} from 'react';
 import { useForm } from 'react-hook-form';
-import { useNavigate  } from 'react-router-dom';
-
+import { useNavigate } from 'react-router-dom';
 import axios from 'axios';
-
 import EyeInvisible from '../../../../authorization/EyeInvisible';
 import EyeVisible from '../../../../authorization/EyeVisible';
 import styles from './SignUpFormContent.module.css';
-
+import PropTypes from 'prop-types';
+import RulesModal from './RulesModal';
```

Comment:
When you use `Suspense` you need to use `React.lazy`

```suggestion
const RulesModal = React.lazy(() => import('./RulesModal'));
```

Reply:
Done